### PR TITLE
feat: migrate user recipes to new structure & format

### PR DESCRIPTION
**Pull request by toph-allen**
Gave them different degrees of copy editing based on need.

### CODE REVIEW

File: 02-users/09-creating-a-user-using-a-remote-authentication-provider-ldap/index.qmd [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Creating a User using LDAP
+title: Creating a User Using a Remote Authentication Provider (LDAP)
```
Comment (tdstein) [inline 02-users/09-creating-a-user-using-a-remote-authentication-provider-ldap/index.qmd:2]: This has been causing me confusion. Is LDAP a specific type of remote authentication provider, or is LDAP the catch-all name for remote authentication providers? If it is the former, should we have a separate recipe for Google OAuth? They would say the same thing more or less, but we can tailor it to the specific authentication provider.

File: 02-users/09-creating-a-user-using-a-remote-authentication-provider-ldap/index.qmd [hunk 2]
```diff
@@ -17,10 +17,10 @@ Obtain a *temporary ticket* using the *remote users* API and then create the use
 ```{.python}
 from posit import connect
 
-client = connect.Client()
-
 USERNAME = "francism"
 
+client = connect.Client()
+
 client = connect.Client()
```
Comment (tdstein) [inline 02-users/09-creating-a-user-using-a-remote-authentication-provider-ldap/index.qmd:24]:
```suggestion
client = connect.Client()
```

**Review by tdstein — APPROVED** [review]

File: 02-users/09-creating-a-user-using-a-remote-authentication-provider-ldap/index.qmd (hunk 1 above)
Comment (toph-allen) [reply 02-users/09-creating-a-user-using-a-remote-authentication-provider-ldap/index.qmd:2]: I _think_ it’s a type of provider — a specific protocol. But now you’ve introduced doubt into my mind 😂

But I think Connect’s API abstracts the differences away. I don't know if remote group lookup is restricted only to some providers or not. I don’t think it is.

Comment (toph-allen) [reply 02-users/09-creating-a-user-using-a-remote-authentication-provider-ldap/index.qmd:2]: I would be happy to remove LDAP from the titles for now and do some investigation as follow up.

Comment (tdstein) [reply 02-users/09-creating-a-user-using-a-remote-authentication-provider-ldap/index.qmd:2]: I'm fine with leaving it as is.

It's very confusing, in my opinion. The "remote" endpoints support LDAP and Google OAuth2, but all other OAuth2 are supported by the "local" endpoints.

**Pull request merged**